imports/lib/schemas/typedSchemas.ts

```diff
@@ -1,9 +1,21 @@
 import { Meteor } from 'meteor/meteor';
 import * as t from 'io-ts';
 import { date } from 'io-ts-types';
-import SimpleSchema, { SchemaDefinition, SimpleSchemaDefinition } from 'simpl-schema';
+import SimpleSchema from 'simpl-schema';
+import type { FieldInfo, SchemaDefinitionWithShorthand, TypeDefinitionProps } from 'simpl-schema/dist/esm/types';
```

Comment:
Ahhhh, I struggled to figure out how to phrase this to get access to the types, since that file isn't actually packaged by the `package.json` -- good to know

Reply:
I just dug around with auto-complete until I found it :joy: